extensions/wikia/CategoryPage3/CategoryPage3Hooks.class.php

```diff
@@ -143,7 +146,7 @@ public static function onLinkerMakeExternalLink( string $url, string $text, stri
 		$categoryNsText = $wgContLang->getNsText( NS_CATEGORY );
 
 		// Not a category
-		if ( strpos( $path, $categoryNsText ) === false ) {
+		if ( strpos( $path, "${$categoryNsText}:" ) === false ) {
```

Comment:
One too many `$`. Also, can `$path` be URL encoded here? If so, should we account for any case in which `:` might be `%3A` (maybe just by unencoding `$path` for the comparison)?